docs/java-grader/index.md

```diff
@@ -158,17 +158,21 @@ By default the Java compiler will show all compilation warnings to the user, exc
     "image": "prairielearn/grader-java",
     "timeout": 10,
     "entrypoint": "autograder.sh",
-    "environment": { "JDK_JAVAC_OPTIONS": "-Xlint:-static -Xmaxerrs 3" }
+    "environment": { "JDK_JAVAC_OPTIONS": "-Xlint:-static -Xmaxerrs 3", "JDK_JAVA_OPTIONS": "-ea" }
   }
 }
 ```
 
-The example above disables the `static` warning (use of static fields applied to object expressions) and limits the number of errors to 3. A more comprehensive list of options can be found in the [`javac` documentation page](https://docs.oracle.com/en/java/javase/11/tools/javac.html). Some options of interest may include:
+The example above disables the `static` warning (use of static fields applied to object expressions) and limits the number of errors to 3. A more comprehensive list of options can be found in the [`javac` documentation page](https://docs.oracle.com/en/java/javase/21/docs/specs/man/javac.html#options). Some options of interest may include:
 
 - `-Xlint:none` or `-nowarn` to disable all warnings;
 - `-Xdoclint` to enable warnings for javadoc comments;
-- `-source 10` to compile using the Java 10 language version;
-- `-ea` to enable [Java assertions](https://docs.oracle.com/javase/7/docs/technotes/guides/language/assert.html).
+- `-source 10` to compile using the Java 10 language version.
+
+Similarly, you may set specific options to the `java` command line using the `JDK_JAVA_OPTIONS`. A list of valid options can be found in the [`java' documentation page](https://docs.oracle.com/en/java/javase/21/docs/specs/man/java.html#standard-options-for-java). Some options of interest may include:
+
+- `-Dproperty=value` to set system properties that may be retrieved with `System.getProperty(name)`;
+- `-ea` to enable [Java assertions](https://docs.oracle.com/javase/8/docs/technotes/guides/language/assert.html).
```

Comment:
This was an error in the documentation. `-ea` is an option of `java` (the VM runner), not `javac` (the compiler).